### DISCUSSION TITLE
Error message when uploading data via Hetztner STORAGE BOX

### DISCUSSION THREAD

**Discussion by MANFahrer-GF** [nextcloud/all-in-one#4300] [Questions] [answered]
Error message when uploading data via Hetztner STORAGE BOX
### Steps for reproducing
1. go to the "Logging" page 
2. there you will see the error message 
3. rmdir(/mnt/ncdata/kantthomas/uploads/web-file-upload-e69e4d222e3f8380): Directory not empty at /var/www/html/lib/private/Files/Storage/Local.php#157

### Expected behavior 
Data should upload without errors ( via web interface)

### Actual behavior 
When data is uploaded via the browser an error message is written e.g.: rmdir(/mnt/ncdata/kantthomas/uploads/web-file-upload-e69e4d222e3f8380): Directory not empty at /var/www/html/lib/private/Files/Storage/Local.php#157

### Host OS 
Ubuntu 22.04

#### Nextcloud AIO version 
AIO v7.13.0 / NextCloud 28.0.2
#### Current channel 
normal
#### Further valuable information 

The STORAGE BOX box is included in the /etc/fstab like this 
//u39XXXX.your-storagebox.de/backup /mnt/storagebox cifs rw,mfsymlinks,seal,credentials=/etc/storage-credentials,uid=33,gid=0,file_mode=0770,dir_mode=0770 0 0

Error message: 

[PHP] Fehler: rmdir(/mnt/ncdata/kantthomas/uploads/web-file-upload-e69e4d222e3f8380): Directory not empty at /var/www/html/lib/private/Files/Storage/Local.php#157
	MOVE /remote.php/dav/uploads/kantthomas/web-file-upload-e69e4d222e3f8380/.file
	von 94.31.103.61 von kantthomas um 01.03.2024, 11:47:52

**Comment by MANFahrer-GF** [comment]
sudo -u www-data ls -lisa /mnt/storagebox
total 4
                  34 0 drwxrwx--- 2 www-data root    0 Feb 18 18:13 .
                  20 4 drwxr-xr-x 4 root     root 4096 Feb 18 19:12 ..
11449930454141650016 0 drwxrwx--- 2 www-data root    0 Mar  1 03:04 ncdata

**Comment by szaimen** [comment, marked as answer]
Hi, I guess you are running into this: https://github.com/nextcloud/all-in-one/discussions/2850#discussioncomment-6269418
